code/datums/perks/fate.dm

```diff
@@ -135,10 +135,25 @@
 		holder.stats.removePerk(src.type)
 		return
 	var/turf/T = get_turf(holder)
-	var/obj/item/W = pickweight(list(
+	var/obj/item/W = null
+	if(is_neotheology_disciple(holder))
+		W = pickweight(list(
+				/obj/item/tool/sword/nt/longsword = 0.5,
+				/obj/item/tool/sword/nt/shortsword = 0.5,
+				/obj/item/tool/sword/nt/scourge = 0.1,
+				/obj/item/tool/knife/dagger/nt = 0.8,
+				/obj/item/gun/energy/nt_svalinn = 0.4,
```

Comment:
Svalinn is just a lesser martin. Consider an old NanoTrasen weapon instead?

Reply:
I meant a ballistic one, instead of the svalinn.
Eitherways, the svalinn doesn't feel like a weapon Noble should give.

Reply:
hmm, alright